game.py

```diff
@@ -0,0 +1,252 @@
+"""
+Created on Thursday Oct 15, 2017
+SkyRoads: MiniProject 4
+
+@authors: Felix Eberhardt & Shreya Rangarajan
+"""
+from threading import Timer
+import time
+import random
+import os, sys, time
+import pygame
+from pygame.locals import *
+
+if not pygame.font:
+    print('Warning, fonts disabled')
+
+running = True
+speed = 3
+
+"""
+Define all the inital variables
+"""
+white = (255, 255, 255)
+black = (0, 0, 0)
+green = (0, 200, 0)
+red = (255, 0, 0)
+green2 = (0, 255, 0)
+blue = (0, 0, 255)
+brown = (165, 42, 42)
+
+car = "audi_1.png"
+barrier = "concrete.png"
+road = "road.png"
+barrier2 = "concrete.png"
+
+display_width = 1024
+display_height = 750
+
+background_size = (display_width, display_height)
+car_size = (100, 67)
+
+"""
+Initialize the game
+"""
+pygame.init()

```

Comment:
Code at this point should be in functions. Then you can name the function, which makes the code more readable. For example, if the function is called `initialize_game` or `init_game` or `game_init`, then it's evident what it does even without the comment. Also, this gives you a place to attach the docstring (and examples!), if it is still useful.